src/PerformanceTests/Persistence.V6.Sql.V2/SqlProfile.cs

```diff
@@ -16,4 +21,14 @@ public void Configure(EndpointConfiguration configuration)
         var subscriptions = persistence.SubscriptionSettings();
         subscriptions.CacheFor(TimeSpan.FromSeconds(1));
     }
+
+    void ISetup.Setup()
```

Comment:
Why not explicitly implement the interface?

Reply:
I've created a separate issue for this https://github.com/Particular/EndToEnd/issues/292